plugins/rule_engines/CMakeLists.txt

```diff
@@ -11,6 +11,50 @@ set(
   passthrough
 )
 
+# messy messy messy
```

Comment:
While *I* think it's funny to leave this in here, if there's nothing actionable (i.e. TODO with an issue number) I'm not sure that we should. :) I defer.

Reply:
Yes, please replace with a TODO + issue number or remove.

Reply:
Pretty sure we already have an open issue about this but I don't feel like tracking it down. Removing.